samer/css/style.css

```diff
@@ -0,0 +1,331 @@
+}
+/* ---------------------------ending menu nav -----------------------*/
+
+
+.main-img {
+    width: 100%;
+    padding-top: 15px;
+    box-shadow: 10px 10px 10px rgba(0,0,0,0.3);
+    margin-top: 50px;
+    
+  }
+
+h1 {
+    text-align: center;
+    padding-bottom: 10px;
+    font-family: Gugi;
+    
+}
+/* ----------------------------history section---------------------- */
+.info{
+font-size: 90%;
+color: #999;
+
+}
+.line {
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+
+.content{
+      padding-top: 60px;
+      overflow: auto; 
+
+  }
+
+
+
+ .his-img{
+     float: left;
+    margin-right: 10px;
+    border-radius: 5%;
+  }
+  #text{
+    text-indent: 20px;
+      text-align: justify;
+      line-height: 1.4;
+  }
+
+ /* ----------show more starting-------------- */
+  .read-more-state {
+    display: none;
+  }
+  
+  .read-more-target {
+    opacity: 0;
+    max-height: 0;
+    font-size: 0;
+    transition: .25s ease;
+  }
+  
+  .read-more-state:checked ~ .read-more-wrap .read-more-target {
+    opacity: 1;
+    font-size: inherit;
+    max-height: 999em;
+  }
+  
+  .read-more-state ~ .read-more-trigger:before {
+    content: 'Show more';
+  }
+  
+  .read-more-state:checked ~ .read-more-trigger:before {
+    content: 'Show less';
+  }
+  
+  .read-more-trigger {
+    cursor: pointer;
+    display: inline-block;
+    color: #666;
+    font-size: .9em;
+    line-height: 2;
+    border: 1px solid #ddd;
+    border-radius: .25em;
+
+  }/* ---------show more ending-------------- */
+
+  /* ---------------------------------map----------------------- */
+  .maps{
+    width: 100%;
+    margin: auto;
+
+  } /* -----------------------ending map-------------------- */
+
+  .images{
+    width: 39%;
+    height: 300px;
+    border-radius: 5%;
+    margin: 2%;
+    display: inline;
+  }
+.cont-img{
+text-align: center
+}
+
+
+/* ----------------------------sights section---------------------- */
+.sights{
+    width: 25%;
+    height: 200px;
+    display: inline;
+    border-radius: 10px;
+    border: solid black 1px;
+    }
+
+    /* ----------------------------gallery section---------------------- */
+.gal{
+
+    width: 800px;
+    height: 535px;
+    float: left;
+    margin: 0px;
+    padding: 0px;
+}
+.slide-container{
+  width: 800px;
+  height: 600px;
+  margin-left: auto;
+  margin-right: auto;
+  
+  overflow: hidden;
+  text-align: center;
+
+}
+.img-container{
+  width: 15200px;
+  height: 600px;
+  position: relative; 
+  transition: left 2s;
+  -webkit-transition: left 2s;
+  -moz-transition: left 2s;
+  -o-transition: left 2s;
+}
+.button-cont{
+  top: -85px;
+  position: relative;
+
+}
+
+.slider-button{
+  display: inline-block;
+  height: 11px;
+  width: 11px;
+  border-radius: 5px;
+  background-color: red;
+
+}
+
+#img1:target ~ .img-container{
+  left: 0px;
+}
+
+#img2:target ~ .img-container{
```

Comment:
All these left: -800px they look fishy as well.
What happens if your images change size, or if your website change size, then you need to redo all this?
We need to find another solution that is more generic and that can adapt when things change.